website/static/js/fangorn.js

```diff
@@ -642,7 +642,7 @@ function _fangornActionColumn (item, col) {
 
     // Upload button if this is a folder
     // If File and FileRead are not defined dropzone is not supported and neither is uploads
-    if (window.File && window.FileReader && item.kind === 'folder' && item.data.provider && item.data.permissions.edit) {
+    if (window.File && window.FileReader && item.kind === 'folder' && item.data.provider && item.data.permissions.edit && !item.data.permissions.diskSavingMode) {
```

Comment:
Talked with @sloria. Binding it to every item rather than to treebeard options give you the option to disable the only the addon u selected without affecting the other addon's behavior.